profiles/api.py

```diff
@@ -38,3 +45,20 @@ def get_channels(user):
             )
         )
     )
+
+
+def get_site_type_from_url(url):
+    """
+    Gets a site type (as defined in profiles.models) from the given URL
+
+    Args:
+        url (str): A URL
+
+    Returns:
+        str: A string indicating the site type
+    """
+    no_fetch_extract = tldextract.TLDExtract(suffix_list_urls=False)
+    extract_result = no_fetch_extract(url)
+    if extract_result.domain in SITE_TYPE_OPTIONS:
```

Comment:
Change to `extract_result.domain.lower()`: if I enter 'HTTP://WWW.FACEBOOK.COM' as my website then it's assigned a `site_type` of 'personal'

Reply:
Great catch 👍